### DISCUSSION TITLE
Projects don't sync across computers

### DISCUSSION THREAD

**Discussion by Kamilkopec** [marcusolsson/obsidian-projects#696] [Help]
I have vault setup on google drive an I synch it between macbook and windows machines.
What I noticed that projects setup'ed in mac don't sync via google drive to windows machine.
Other stuff synching correctly, no issues.
Tried to re-open and open the vaults again on both machines, no effect.
Anything I'm doing wrong?

**Comment by Acylation** [comment]
Hello! The plugin saves the projects information in the `data.json` file, which is located within the config folder (typically `.obsidian` and depending on your Obsidian preferences).  If your sync service is set to exclude the config folder, the project data will not be synchronized. Would setting up rules within your sync service to allow this specific file be of help?

**Reply by Kamilkopec** [reply]
Thanks. After looking over the synch logs for gd I noticed that this file was stuck with error.
Thanks for pushing in the right direction
